neqo-qpack/src/decoder.rs

```diff
@@ -160,7 +169,10 @@ impl QPackDecoder {
         }
     }
 
-    // this function returns None if the stream is blocked waiting for table insertions.
+    /// This function returns None if the stream is blocked waiting for table insertions.
+    /// buf must contain the complete header block.
```

Comment:
I read this as "but" at first.

```suggestion
    /// `buf` must contain the complete header block.
```